x/stakeibc/keeper/trade_route_test.go

```diff
@@ -0,0 +1,119 @@
+package keeper_test
+
+import (
+	"strconv"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	keepertest "github.com/Stride-Labs/stride/v14/testutil/keeper"
+	"github.com/Stride-Labs/stride/v14/testutil/nullify"
+	"github.com/Stride-Labs/stride/v14/x/stakeibc/keeper"
+	"github.com/Stride-Labs/stride/v14/x/stakeibc/types"
+)
+
+func createNTradeRoute(keeper *keeper.Keeper, ctx sdk.Context, n int) []types.TradeRoute {
+	items := make([]types.TradeRoute, n)
+	for i := range items {
+
+		hostChain := strconv.Itoa(i) + "chain"
```

Comment:
I know we have a lot of tests scaffolded this way cause of ignite, but I don't think there's really any value in filling out the fields that aren't the keys, and it makes it a bit annoying every time we modify fields